app/views/users/show.html.erb

```diff
@@ -1 +1,39 @@
-<h1>show all the users</h1>
\ No newline at end of file
+<div class = 'users-tab'>  
+  <div class = 'users-header'>
+    <div class = 'users-row-info'>
+      <div class = 'img-container'>
+          <div class="inner-border">
+            <img src="<%= @user.photo %>" alt="" width="100" height="100">
+          </div>
+        </div>
+      <h1><%= @user.name %></h1>
+    </div>
+    <div>
+      <h3>
+        Posts: <%= @user.post_counter %>
+      </h3>
+    </div>
+  </div>
+  <div>
+    <h3>
+      Bio: <%= @user.bio %>
+    </h3>
+  </div>
+  <div>
+    <h3>Posts: </h3>
+    <ul>
+      <% @user.recent_posts.each do |post| %>
+        <li><%= post.title %></li>
+      <% end %>

```

Comment:
- [ ] according to the requirements the post text, the comment counter and the like counter for each post should be visible as well.